src/lib/nodes/eth/EthNode.ts

```diff
@@ -32,4 +36,16 @@ export class EthNode extends Node {
       ping: Date.now() - time
     }
   }
+
+  private async fetchNodeVersion(): Promise<void> {
+    try {
+      const parts = (await this.client.getNodeInfo()).split('/')
+      const clientName = parts.length > 0 ? parts[0] : ''
+      const fullVersion = parts.length > 1 ? parts[1]: ''
+      const simplifiedVersion = fullVersion.split('-')[0]
+      this.version = `${clientName}/${simplifiedVersion}`
```

Comment:
Create a helper function for normalizing the version, as you did for BTC `formatBtcVersion()`